Shadowlands/TazaveshTheVeiledMarket/Trash.lua

```diff
@@ -0,0 +1,333 @@
+	self:Log("SPELL_CAST_START", "InvigoratingFishStick", 355132)
+	self:Log("SPELL_CAST_SUCCESS", "InvigoratingFishStickSpawned", 355132)
+	self:Log("SPELL_CAST_START", "CryofMrrggllrrgg", 355057)
+	self:Log("SPELL_AURA_APPLIED", "CryofMrrggllrrggApplied", 355057)
+	self:Log("SPELL_CAST_START", "TidalStomp", 355429)
+	self:Log("SPELL_CAST_START", "ChargedPulse", 355584)
+	self:Log("SPELL_CAST_START", "Crackle", 355577)
+	self:Log("SPELL_AURA_APPLIED", "CrackleDamage", 355581)
+	self:Log("SPELL_PERIODIC_DAMAGE", "CrackleDamage", 355581)
+	self:Log("SPELL_PERIODIC_MISSED", "CrackleDamage", 355581)
+	self:Log("SPELL_CAST_START", "SuperSaison", 356133)
+	self:Log("SPELL_AURA_APPLIED", "SuperSaisonApplied", 356133)
+	self:Log("SPELL_CAST_START", "DriftingStar", 357226)
+	self:Log("SPELL_CAST_SUCCESS", "WanderingPulsar", 357238)
+	self:Log("SPELL_CAST_START", "UnstableRift", 357260)
+	self:Log("SPELL_CAST_START", "Reinvigorate", 357284)
+end
+
+--------------------------------------------------------------------------------
+-- Event Handlers
+--
+
+------ Streets of Wonder ------
+
+-- Interrogation Specialist
+function mod:StasisBeam(args)
+	self:Message(args.spellId, "yellow", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "alert")
+end
+
+-- Armored Overseer / Tracker Zo'korss
+function mod:BeamSplicer(args)
+	self:Message(args.spellId, "cyan", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "warning")
+end
+do
+	local prev = 0
+	function mod:BeamSplicerDamage(args)
+		if self:Me(args.destGUID) then
+			local t = args.time
+			if t-prev > 1 then
+				prev = t
+				self:PersonalMessage(356001, "underyou")
+				self:PlaySound(356001, "underyou")
+			end
+		end
+	end
+end
+
+-- Cartel Muscle
+function mod:HyperlightBackhand(args)
+	self:Message(args.spellId, "yellow", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "alert")
+end
+
+-- Cartel Smuggler
+function mod:HyperlightBombApplied(args)
+	if self:Me(args.destGUID) then
+		self:PersonalMessage(args.spellId)
+		self:PlaySound(args.spellId, "alarm")
+		self:SayCountdown(args.spellId, 5)
+	end
+end
+function mod:HyperlightBombRemoved(args)
+	if self:Me(args.destGUID) then
+		self:CancelSayCountdown(args.spellId)
+	end
+end
+
+-- Support Officer
+function mod:RefractionShieldApplied(args)
+	self:Message(args.spellId, "yellow", CL.on:format(args.spellName, args.destName))
+	self:PlaySound(args.spellId, "warning")
+end
+function mod:HardLightBarrier(args)
+	self:Message(args.spellId, "orange", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "alert")
+end
+function mod:HardLightBarrierApplied(args)
+	if self:Dispeller("magic", true, args.spellId) then
+		self:Message(args.spellId, "cyan", CL.buff_other:format(args.destName, args.spellName))
+		self:PlaySound(args.spellId, "warning")
+	end
+end
+
+-- Commerce Enforcer
+function mod:ForceMultiplierApplied(args)
+	if self:Tank() or self:Healer() or self:Dispeller("enrage", true) then
```

Comment:
I haven't played the game in a while so I don't know how groups usually handle Force Multiplier, but if it's popular to let this cast go through and not soothe, DPS may want a notification for this as well for burst damage. It'll be your call here.

Reply:
It's a very important soothe, probably the most dangerous enrage effects in the current set of dungeons. So I can see the argument for notifying non-soothing DPS when it goes out.